sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,254 @@
+- Moved [`sycl_ext_oneapi_memcpy2d`](doc/extensions/supported/sycl_ext_oneapi_memcpy2d.asciidoc) to supported [751acf35]
+- Updated documentation for `-fsycl-no-rdc` mode [a27e6fc5]
+- Added an initial draft of non-uniform sub-groups proposal [cc7cbd28] 
+[a5d62c8a]
+- Added a draft of 16-bit atomics extension [1b973aed]
+- Added an extension specification for USM allocation functions with properties 
+support [3600f922]
+- Added `sycl_ext_oneapi_peer_access` extention specification [b5bce772] 
+[37960cda]
+
+## Improvements
+
+### SYCL Compiler
+- Improved `sycl::reqd_work_group_size` for optional dimensions [b211d0df]
+- Enabled DPC++ build with llvm-mingw toolchain on Windows [1715c36b]
+- Deprecated usage of `-sycl-std=1.2.1` [8dd6fdff]
+- Disabled force inlining of kernel call operator for FGPA [90e0e650]
+- Reimplemented `-f[no]sycl-early-optimizations` flag [d164fd94]
+
+### SYCL Library
+- Started to use const qualifier in several accessor iteratoration operators 
+[e64433c9]
+- Added diagnostics that the joint_matrix API is only supported by the Intel and 
+CUDA backends [ed596eca]
+- Deprecated `sycl::ext::oneapi::leader(group)`, SYCL 2020 member function 
+`group::leader` should be used instead [94434e78]
+- Deprecated `get_access` for `sycl::host_accessor` in accordance with SYCL 2020 
+[d145da8f]
+- Deprecated `get_size()` and `get_max_statement_size()` member functions in the 
+`sycl::stream` class, added replacements: `size()` and `get_work_item_buffer_size()` [aab0ba71]
+- Deprecated legacy scalar and vector type aliases [e83d0ae0] [016d68b0]
+- Empty platforms now prohibited to be returned [5e8ae211]
+- Started to throw an exception when selected device is not in context 
+[262299d1]
+- Changed signature of `sycl::host_accessor::get_pointer` in accordance with 
+SYCL 2020 [a7473fd2]
+- Started to ignore placeholder template parameter for accessors in accordance 
+with SYCL 2020 [2b1d3205]
+- Started to throw the correct exception when passing unbound accessor to 
+command [22cefabf]
+- Hid the members of `sycl::reducer` that are not mentioned in the SYCL 2020 
+specification and introduced the identity member function [505aa7d1]
+- Improved `group_local_memory_for_overwrite` (part of 
+[`sycl_ext_oneapi_local_memory`](doc/extensions/supported/sycl_ext_oneapi_local_memory.asciidoc)) 
+by defailt initializing it [37a43f93]
+- Augmented `sycl::has_known_identity` to return true for `std::complex` and 
+`std::plus` operators [2e733125]
+- Improved diagnostic when `sycl::accessor` isn't bound to `sycl::handler` 
+[3c87f55e]
+- Added identities for arithmetic and integral operations on bool in accordance 
+with the SYCL 2020 specification, added a check for the specification defined 
+identities [091481d8]
+- Renamed `raw_send{s}_{load/store}` APIs in ESIMD extension [cb280eab]
+- Allowed const `sycl::ext::oneapi::experimental::device_global` [a43c5bd1]
```

Comment:
Tagging @steffenlarsen to review this line

Reply:
Same as my other device_global comment.